tracer/src/Datadog.Trace/ClrProfiler/ServerlessInstrumentation/ServerlessMiniAgentManager.cs

```diff
@@ -0,0 +1,91 @@
+// <copyright file="ServerlessMiniAgentManager.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Diagnostics;
+using Datadog.Trace.Logging;
+
+namespace Datadog.Trace.ClrProfiler.ServerlessInstrumentation;
+
+internal class ServerlessMiniAgentManager
```

Comment:
It looks like this is also stateless
```suggestion
internal static class ServerlessMiniAgentManager
```

Reply:
fixed